bin/update-test-jquery-updates.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+if [ $# -gt 1 ]; then
+  echo "Syntax: $0 <revision/branch>"
+  echo
+  echo "Revision can be left blank, which will update to the top of trunk branch"
+  echo
+  echo "Example: $0"
+  echo "This will update the test-jquery-updates plugin to latest code"
+  exit 1
+fi
+
+print_heading()
+{
+  echo "\n* $1"
+}
+
+revision="trunk"
+if [ "$1" ]; then
+  revision=$1
+fi
+
+directory="test-jquery-updates"
+
+if [ -d "$directory" ]; then
+  print_heading "Removing $directory directory before update"
+  rm -rf $directory
```

Comment:
Because we need to mess with the code (see lower about `install_plugins`), the git history gets all messed up. So I end up removing the folder and recreating on every update